data/crac-creation/crac-creator-cim/src/main/java/com/powsybl/openrao/data/craccreation/creator/cim/craccreator/NetworkActionCreator.java

```diff
@@ -127,13 +128,13 @@ private void addPstSetpointElementaryAction(String elementaryActionId, RemedialA
         } else {
             throw new OpenRaoImportException(ImportStatus.INCONSISTENCY_IN_DATA, String.format("Wrong marketObjectStatusStatus: %s on elementary action %s", marketObjectStatusStatus, elementaryActionId));
         }
-        networkActionAdder.newPstSetPoint()
+        networkActionAdder.newPhaseTapChangerTapPositionAction()
             .withNetworkElement(networkElementId)
-            .withSetpoint(setpoint)
+            .withTapPosition((pstHelper.getLowTapPosition() + pstHelper.getHighTapPosition()) / 2 + setpoint)
```

Comment:
We have in the API a method `getNeutralPosition()` that gives the tap where rho = 1 and alpha = 0 if exists. What is the initial need?

Reply:
Ok, you are just iso-functional with this line.